.pipelines/pipeline.user.windows.yml

```diff
@@ -10,7 +10,6 @@ restore:
     - !!defaultcommand
       name: 'Restore'
       command: 'build.cmd'
-      arguments: 'RestoreOnly'
```

Comment:
Are we calling build.cmd twice, once for restore and once for build? Do we still need this restore section?

Reply:
Yes, this was done because pulling nuget packages can only be done in the "Restore" phase of cdpx builds. Invoking build during the restore phase caused the nuget packages to get pulled and then cached for the rest of the build.

Although, building twice is unnecessary so I have just restored the "-RestoreOnly" argument and instead updated the build.ps1 script to respect it.